nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -18,6 +18,8 @@
 
 - [sitespeed-io](https://sitespeed.io), a tool that can generate metrics (timings, diagnostics) for websites. Available as [services.sitespeed-io](#opt-services.sitespeed-io.enable).
 
+- [Jool](https://nicmx.github.io/Jool/en/index.html), an Open Source implementation of IPv4/IPv6 translation on Linux. Available as [networking.jool.enable](#opt-networking.jool.enable).
```

Comment:
I just used what the project calls itself: https://nicmx.github.io/Jool/en/intro-jool.html

Reply:
And on the homepage they say

> Jool is an Open Source [SIIT and NAT64](https://nicmx.github.io/Jool/en/intro-xlat.html) for Linux.

I don't think what they say on the intro page is very clear, but it is fine *there*, since you're already on their homepage. I consider it somewhat like slang, that you can use inside a group that is familiar.

The NixOS release notes likely aren't that. From my experience, I can say that this would get rewritten by the release editors before release. Ideally, we can do that now, since we have more context.